### ISSUE TITLE
Get compilation results on deplyment

### ISSUE THREAD

**Issue by max-degterev** [techpines/asset-rack#44]
Hi!

AssetRack is awesome for development, but I cannot figure a best way to get compiled results on deployment. I wanted to serve static assets with nginx and proxy all other requests to node. I see that AssetRack supports Amazon S3 and other cloud storages, but what if I want to serve resulting assets locally? Can you add a way to get compiled results for deployment, save them to a disk or something?

**Comment by techpines** [comment]
Alright, this is finally done:

https://github.com/techpines/asset-rack/blob/master/README.md#running-in-production-mode

**Comment by max-degterev** [comment]
Awesome! Will test today on my project! Thanks!

**Comment by max-degterev** [comment]
Im sorry still not getting it. I'm proxying my public folder through nginx to make it faster. I thought I could write results of compilation to disk somewhere. Preferably in the public folder :)

**Comment by techpines** [comment]
Ok, you want to write the assets to disk.  It's coming.

**Comment by max-degterev** [comment]
Yup exactly. Right now we came up with this: https://gist.github.com/suprMax/5101852
as a temporary measure. Would be nice to see that working out of the box
